structure/validator/validator.go

```diff
@@ -94,6 +94,10 @@ func (v *Validator) Array(reference string, value *[]interface{}) structure.Arra
 	return NewArray(v.base.WithReference(reference), value)
 }
 
+func (v *Validator) Bytes(reference string, value []byte) structure.Bytes {
```

Comment:
Don't get me wrong, consistency is fine, but making the code more complicated just so that a bunch of functions all have an argument whose type starts with `*` is not a worthwhile tradeoff for me.

Reply:
Disagree, but your code, your choice.